ntfysh_client/NotificationDialog.cs

```diff
@@ -0,0 +1,263 @@
+﻿using System;
+using System.Drawing;
+using System.Runtime.InteropServices;
+using System.Windows.Forms;
+using System.ComponentModel;
+using Microsoft.Win32;
+using System.Diagnostics;
+
+
+namespace ntfysh_client
+{
+    public partial class NotificationDialog : Form
+    {
+        [DllImport("user32.dll", CharSet = CharSet.Auto)]
+        private static extern bool AnimateWindow(IntPtr hWnd, int time, int flags);
+
+        private const int ScreenMargin = 20;
+
+        private int _timeout = 0;
+        private System.Timers.Timer? displayTimeoutTimer = null;
+        private System.Windows.Forms.Timer? updateTimer = null;
+        private Stopwatch? shownStopwatch = null;
+        private ToolTipIcon? _icon;
+        private int _progress_value = 0;
+        private int progress
+        {
+            get { return this._progress_value; }
+            set
+            {
+                this._progress_value = value;
+                this.progressBar1.Value = value;
+            }
+        }
+
+        public bool IsVisible
+        {
+            get { return this.Visible; }
+            set { this.Visible = value; }
+        }
+
+        public NotificationDialog()
+        {
+            this.ShowInTaskbar = false;
+            this.IsVisible = false;
+            this.TopMost = true;
+            InitializeComponent();
+            InitializeWindowHidden();
+        }
+
+        public void ShowNotification(string title, string message, int timeout_ms = -1, ToolTipIcon? icon = null, bool showTimeOutBar = true, bool showInDarkMode = true)
+        {
+            if (this.IsVisible)
+            {
+                // close the current notification
+                this.handleTimeout(null, null);
+            }
+            // setup data
+            this._icon = icon;
```

Comment:
Nitpick: newline

Reply:
I added a blank line before the if statement